File: Upcoder/Other/Ba cạnh tam giác/LTCB_GK.Cau5.cs
using System;
namespace Tamgiac
{
    public class Program
    {
        static void Main(string[]args)
        {
            string [] input = Console.ReadLine().Split(' ');
            int a = int.Parse(input[0]);
            int b = int.Parse(input[1]);
            int c = int.Parse(input[2]);
            if (a > 0 && b > 0 && c > 0 && a + b > c && a + c > b && b + c > a)
            {
                double p = (a + b + c) / 2.0;
                double S = Math.Sqrt(p * (p - a) * (p - b) * (p - c));
                int perimeter = a + b + c;
                Console.Write("{0} ", perimeter);
                Console.Write("{0}", (int)S);
            } 
            else 
                Console.Write("Khong phai tam giac");
        }
    }
}

File: Upcoder/Other/Ba cạnh tam giác/LTCB_GK.Cau5.py
import math
a, b, c = map(int , input().split())
if (a > 0 and b > 0 and c > 0 and a + b > c and a + c > b and b + c > a): 
    p = (a + b + c) / 2.0
    S = math.sqrt(p * (p - a) * (p - b) * (p - c))
    perimeter = a + b + c
    print(perimeter,int(S)) 
else:
    print("Khong phai tam giac", end="")